lib/oelite/fetch/git.py

```diff
@@ -267,6 +267,17 @@ def unpack(self, d):
         return True
 
     def mirror(self, mirror=os.getcwd()):
+        if self.is_local:
+            # Don't mirror if the source is local. Concept taken from fetch
+            # Do check revision exists before accepting though
+            print "Warning: Skipping creating mirror for local path %s"%(self.uri)
```

Comment:
Dictator decision: Replace the 'Warning:' prefix with something like 'Info:' instead.

Usecase: You have created a recipe with a local git SRC_URI and execute a mirrorall task. You haven't specifically requested to mirror the local git SRC_URI, and actually have no good way to get rid of that warning while doing the perfectly valid mirrorall task.